VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/declaration/declaration.component.html

```diff
@@ -5,20 +5,19 @@
 </div>
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-xl">Declaration</h1>
+    <h1 class="govuk-heading-xl">{{ 'declaration.declaration' | translate }}</h1>
   </div>
 </div>
 <form [formGroup]="declarationForm" class="govuk-form-group" (ngSubmit)="onSubmit()">
   <div [ngClass]="declarationForm.invalid && submitted ? 'govuk-form-group--error' : 'govuk-form-group'">
     <div *ngIf="declarationForm.invalid && submitted" class="govuk-error-message">
-      <span class="govuk-visually-hidden">Error:</span> Please tick the box to confirm that you will not record the hearing in any way
+      <span class="govuk-visually-hidden">{{ 'declaration.error' | translate }}</span> {{ 'declaration.please-tick-the-box-to-confirm-that-you-will-not-record-the-hearing-in-any-way' | translate }}
```

Comment:
could be shortened to 
declaration.confirm-not-recording

Reply:
Shortened